news/_posts/2020-05-13-the-new-order-pages-in-prestaShop-1.7.7.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: post
+title: "The new order pages in PrestaShop 1.7.7"
+subtitle: "Introducing the revamp of order pages"
+date: 2020-05-12 12:00:00
+authors: [MateusShirlaw]
+icon: icon-leaf
+tags:
+- 1.7.7
+- version
+- minor
+- product
+- ux
+---
+
+With the release of the new version 1.7.7, this article has the ambition to detail one of the major features of the version. Our work was focused on one of the most used pages by our merchants: the order page.
+
+We are thrilled to introduce to you the new feature around the order process. But before jumping on the features that await the merchant in the new version, we&#39;d like to share our thinking process.
+
+### ***First, why rethinking this page in particular?***
+
+The page was beginning to be particularly unpleasant to use because of its old visual appearance and its hierarchy of information that led to inefficiency and misunderstanding for our merchants.
+
+### ***A long process to ensure the full success of the revamp.***
+
+To have a better understanding of the merchants&#39; pain points, we have interviewed merchants with different sales volume, country and business activity as well as agencies of different sizes. The pain points encountered have then been addressed by the features added in the order list page (I) and order detail page (II).
+
+### **Now, let's present those changes.**
+
+## **I.  Concerning the page listing the orders, we have added the following features**
+
+**The objective of the new features of the listing order page is to help merchants save time and increase their productivity.**
+
+- **Order's preview**: Opens a panel just below the line of the order to preview the order's essential information.
+
+![Order preview](/assets/images/2020/05/order_preview.gif "Order preview")
+
+***What is the goal***:
+
+Allows the merchant to check, prepare and track his order without having to open it. So the merchant won't have to open the order to check essential information that doesn&#39;t require actions.
+
+***How it works***: 
+
+When the user clicks on the icon, it displays the order&#39;s preview below the line selected. The order preview lists the following information: the invoice address, the shipping address, the customer details, the carrier details (and the tracking ID when available) and the products' list. 
+
+![](RackMultipart20200512-4-1tjtyoa_html_a1644e9702a7fb4f.png)
```

Comment:
💩

Reply:
I stop my review for the moment, I realized you still have some work to do on the article.
Let me know when it's ready :)